packages/cli/src/cmds/dev/handler.ts

```diff
@@ -101,15 +101,16 @@ export async function devHandler(args: IDevArgs & IGlobalArgs): Promise<void> {
 
   if (args.startValidators) {
     const secretKeys: SecretKey[] = [];
-    const [fromIndex, toIndex] = args.startValidators.split(":").map((s) => parseInt(s));
+    const [fromIndex, valCount] = args.startValidators.split(":").map((s) => parseInt(s));
+    const toIndex = valCount - 1;
     const maxIndex = anchorState.validators.length - 1;
 
     if (fromIndex > toIndex) {
       throw Error(`Invalid startValidators arg '${args.startValidators}' - fromIndex > toIndex`);
     }
 
     if (toIndex > maxIndex) {
-      throw Error(`Invalid startValidators arg '${args.startValidators}' - state has ${maxIndex} validators`);
+      throw Error(`Invalid startValidators arg '${args.startValidators}' - state has ${valCount} validators`);
```

Comment:
should be `${valCount - fromIndex}`?
Maybe better to retain the old `toIndex` variable name

Reply:
`const toIndex = valCount - fromIndex - 1;` like this? I was looking for confirmation that the old `toIndex` is actually the validator count. Maybe @dapplion can clarify.

Reply:
I think the `--startValidators` option would be `fromIndex:toIndex`
[fromIndex, toIndex)
including fromIndex and up to, but not including toIndex.